### PR TITLE
PWGPP-576 unit tests

### PR DESCRIPTION
**Pull request by pl0xz0rz**
This PR:
-Adds the pulls test to the Jupyter notebook
-Adds the unit tests to benchmark_bootstrap.py (copied from the Jupyter notebook)
-Fixes bootstrapping for the Tensorflow fitter
-Fixes the weights for single fit TensorFlow test (this improves the fit accuracy but for some reason breaks the error estimate, will have to find out why)

### CODE REVIEW

**Comment by miranov25** [comment]
All tests for pytotch and scikit looks fine.
See JIRA
https://alice.its.cern.ch/jira/browse/PWGPP-576?focusedCommentId=249508&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-249508

|Fit type:|npoints|mean|rms_estimate|status|
|-|-|-|-|-|
|Pytorch_LBFGS|1000|0.000259|0.000450|1|
|Pytorch_LBFGS_CUDA|1000|0.000259|0.000450|1|
|Scipy_LM|1000|0.000259|0.000450|1|
|Tensorflow_BFGS|1000|0.000259|0.001265|1|
|Pytorch_LBFGS|10000|-0.000053|0.000142|1|
|Pytorch_LBFGS_CUDA|10000|-0.000053|0.000142|1|
|Scipy_LM|10000|-0.000053|0.000142|1|
|Tensorflow_BFGS|10000|-0.000053|0.000471|1|
|Pytorch_LBFGS|100000|0.000077|0.000045|1|
|Pytorch_LBFGS_CUDA|100000|0.000077|0.000045|1|
|Scipy_LM|100000|0.000077|0.000045|1|
|Tensorflow_BFGS|100000|0.000077|0.000128|1|
|Pytorch_LBFGS|1000000|0.000007|0.000014|1|
|Pytorch_LBFGS_CUDA|1000000|0.000007|0.000014|1|
|Scipy_LM|1000000|0.000007|0.000014|1|
|Tensorflow_BFGS|1000000|0.000007|0.000048|1|
